force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,9 +32,15 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        
+        if(a != null && b != null){
```

Comment:
Its best practice to handle null values first, then move onto any logic only if your parameters have values.